__tests__/buildPlatform.test.js

```diff
@@ -24,7 +24,7 @@ describe('buildPlatform', () => {
 
   it("should throw if passed a platform that doesn't exist", async () => {
     await expect(StyleDictionaryExtended.buildPlatform('foobar')).to.eventually.rejectedWith(
-      'Platform "foobar" does not exist',
+      'Please supply a valid platform, "foobar" does not exist',
```

Comment:
This "new" error message combines the two old messages, which both meant the exact same thing but were not deduped.